public/js/actions/user.js

```diff
@@ -2,12 +2,24 @@
 
 var $ = require('jquery');
 
-var appActions = require('./app');
 var actionTypes = require('constants/action-types');
+var settings = require('settings');
+
+var appActions = require('./app');
+
+
+function setUpCSRF(csrfToken) {
+  console.log('setting CSRF token for subsequent requests:', csrfToken);
+  $.ajaxSetup({
+    headers: {
+      'X-CSRFToken': csrfToken,
+    },
+  });
+}
 
 
-export function signIn(accessToken) {
-  return function(dispatch) {
+export function tokenSignIn(accessToken) {
+  return dispatch => {
```

Comment:
Yeah, let's see. For me it's more about consistency than anything else. But yes - maybe I get used to it soon enough. It's a brave new world!

![back-to-the-future](https://cloud.githubusercontent.com/assets/1514/8929298/439e30c4-351c-11e5-8dec-77c614081d44.gif)